bazel/linux/defs.bzl

```diff
@@ -400,15 +400,17 @@ def _kernel_module_targets(*args, **kwargs):
 
         kwargs["name"] = name
         kwargs["kernel"] = kernel_tree_label
-        _gen_module_rule(arch, *args, **kwargs)
+        deps = kwargs["deps"] + [kdep.format(kernel) for kdep in kernel_specific_deps]
+        my_kwargs = dict(kwargs, deps=deps)
+        _gen_module_rule(arch, *args, **my_kwargs)
 
     # This creates a target with the name chosen by the user that
     # builds all the modules for all the requested kernels at once.
     # Without this, the user can only build :all, or the specific
     # module for a specific kernel.
     return kernel_modules_bundle(name = original, modules = targets, visibility = kwargs.get("visibility"))
 
-def kernel_module(*args, **kwargs):
+def kernel_module(kernel_specific_deps=[], *args, **kwargs):
```

Comment:
Please update the docstring here.

Reply:
Update for your other comment got rid of `kernel_specific_deps`, but I added a section for `deps` to describe the special interpolation behavior.